packages/core/src/domain/internalMonitoring.ts

```diff
@@ -26,24 +26,29 @@ export interface MonitoringMessage extends Context {
 }
 
 const monitoringConfiguration: {
-  batch?: Batch
   debugMode?: boolean
   maxMessagesPerPage: number
   sentMessageCount: number
 } = { maxMessagesPerPage: 0, sentMessageCount: 0 }
 
 export let externalContextProvider: () => Context
 
+let onInternalMonitoringEventCollected: ((message: MonitoringMessage) => void) | undefined
```

Comment:
Since this function takes a `MonitoringMessage` as argument, I think it should be called `onInternalMonitoringMessageCollected`

Reply:
Hum, why not but we could say the same for log [`onLogEventCollected`](https://github.com/DataDog/browser-sdk/blob/c0356a2b8fb852dba1d40689fbd09217011bf650/packages/logs/src/boot/startLogs.ts#L63).
Should I update both?

Reply:
I think that you could, it's a small enough change to be included in this PR. But feel free to keep the current naming, this is more like nitpicking.